toolkit/recordreplay/ProcessRecordReplay.cpp

```diff
@@ -160,8 +162,9 @@ static void ConfigureGecko() {
   // Don't create a stylo thread pool when recording or replaying.
   putenv((char*)"STYLO_THREADS=1");
 
-  // This mutex needs to be initialized on a consistent thread.
+  // These mutexes need to be initialized at consistent points.
```

Comment:
It is worth linking to https://github.com/RecordReplay/backend/issues/4408 from this comment to provide even more context?

Or maybe write something similar to what you wrote up in https://github.com/RecordReplay/backend/issues/4408#issuecomment-1024920488:

"StaticMutex objects initialize their underlying mutex the first time they are locked. If two threads are racing to do this initialization then it can happen at different points when recording vs. replaying, and we get mismatches that cause replaying failures. To work around this we initialize these mutexes explicitly here so that it happens at a consistent point in time."

Reply:
Sure, I updated the comment.